sec-keys/openpgp-keys-libmicrohttpd/openpgp-keys-libmicrohttpd-999999.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OpenPGP keys used to sign GNU libmicrohttpd releases"
+HOMEPAGE="https://savannah.gnu.org/projects/libmicrohttpd/"
+if [[ ${PV} == 9999* ]] ; then
+	PROPERTIES="live"
+
+	BDEPEND="|| ( net-misc/wget[gnutls] net-misc/wget[ssl] )"
+else
+	KARLSON2K_ID='EA812DBEFA5A7EF17DA8F2C1460A317C3326D2AE'
+	CHRISTIAN_ID='D8423BCB326C7907033929C7939E6BE1E29FC3CC'
+	# Technically public keys are not stable.
+	# While the fingerprint is stable, the contents of the key can be
+	# changed at any time by an additional user ID (email address) or
+	# an additional signature.
+	SRC_URI="https://keys.openpgp.org/vks/v1/by-fingerprint/${CHRISTIAN_ID} -> christiangrothoff-${CHRISTIAN_ID}.asc
+		https://keys.openpgp.org/vks/v1/by-fingerprint/${KARLSON2K_ID} -> karlson2k-${KARLSON2K_ID}.asc"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+fi
+
+S="${WORKDIR}"
+
+LICENSE="public-domain"
+SLOT="${PV}"
```

Comment:
How does this even remotely make sense, when you explicitly require a specific slot from the package?

Reply:
Sorry, I don't understand the question.

Live version resides in the same slot number 999999 and the key in this slot matches the latest version of the `libmicrohttpd` package. The key could be changed, but should always match the latest `libmicrohttpd` version. I don't see what's wrong with this combination.

I can remove this file from this PR if it is a problem.

Reply:
The live version could be used simultaneously with other versions. That's why it has a slot.

Reply:
But no version will ever use the live version because every version will need to depend on non-live version.

Reply:
Except live versions, which could depend on live version of another package.
There are a lot of such packages in the tree, for example [`dev-util/pkgcheck`](https://github.com/gentoo/gentoo/tree/master/dev-util/pkgcheck) or [`dev-util/pkgdev`](https://github.com/gentoo/gentoo/tree/master/dev-util/pkgdev).

That's the idea of this keyring.

Reply:
How are you going to use the live version of the keyring if live versions, quite obviously, don't come with signed release tarballs?

Reply:
To be precise:
I didn't want to implement the keychain for real "live" library version. Actually my idea was to provide a simple way to check whether the upstream keys changed (when signature would fail for the new release) or something wrong with downloaded file. It should be easy to quickly check the new library release against 9999 keys and if it succeed, update the `openpgp-keys-` package.
In practice I don't need this file as I would know how and when the upstream signature keys are updated. It is more for others who will need to update/use these keys in case if I would not update this ebuild for whatever reason.
So I can remove the 9999 version completely, add more comments explaining my idea to the ebuild or comment out "live" part in full (so someone would be able to restore/re-use it for the latest keyring download).

Please let me know what is the best way for you.